core/services/ping/ping360_ethernet_prober.py

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env python3
+"""
+Scan with ./findPing360.py [current IP of the interface to scan]
+"""
+
+import asyncio
+import socket
+from typing import List
+
+import psutil
+
+
+def list_ips() -> List[str]:
+    """
+    Returns a list of IPs found on ethernet interfaces that are currently up
+    """
+    stats = psutil.net_if_stats()
+
+    available_networks = []
+    ips = []
+    for interface in stats.keys():
+        if interface in stats and getattr(stats[interface], "isup"):
+            available_networks.append(interface)
+
+    for interface in available_networks:
+        phys_list = list(filter(lambda address: address.family == socket.AF_INET, psutil.net_if_addrs()[interface]))
+        ips.extend([phys.address for phys in phys_list])
+    return list(set(ips))  # make sure there are not duplicated entries
+
+
+def remove_zeros(ip: str) -> str:
+    """
+    This removes the leading zeros return by the Ping360 firmware.
+    This functions transforms 192.168.015.016 into 192.168.15.16
+    """
+    new_ip = ".".join([str(int(i)) for i in ip.split(".")])
```

Comment:
Use generator expression instead of list comprehension for single-use iterator
```suggestion
    new_ip = ".".join(str(int(i)) for i in ip.split("."))
```